### ISSUE TITLE
Can't reporduce the results on libero_spatial

### ISSUE THREAD

**Issue by dreamcubeblock** [OpenDriveLab/UniVLA#26]
We fine-tuned the model on an A100 GPU with the same basic hyperparameters, except that we modified shuffle_buffer_size to prevent OOM.  In the final evaluation, the accuracy was less than 60%. Could you give me some advice?  Thank you!

![Image](https://github.com/user-attachments/assets/1234ce9e-bc65-4c7c-b74c-f7a271531812)

![Image](https://github.com/user-attachments/assets/50c215b4-7b79-41a9-a8e4-8ce202cf2d83)

**Comment by retsuh-bqw** [comment]
I noticed that you're training on a single GPU, which means your effective batch size is only 1/8 of our original setup while using the same large learning rate. To better align with our training configuration, you might consider:

1. Using multiple GPUs to match our setup (if resources allow), or

2. Reducing the learning rate (e.g., to ~5e-5) and increasing the number of optimization steps to compensate.

**Comment by dreamcubeblock** [comment]
Thank you so much, I will try it right now!
